testcases/PowerNVDump.py

```diff
@@ -305,6 +309,76 @@ def runTest(self):
         return boot_type
 
 
+class SBECrash_MPIPL(PowerNVDump):
+
+    def runTest(self):
+
+        if "OpenBMC" in self.bmc_type:
+            cmd = "mboxctl --reset"
+        elif "SMC" in self.bmc_type:
```

Comment:
Skip test for all platform except OpenBMC for now. (not just SMC)

Reply:
Ok